src/content/it/fundamentals/performance/critical-rendering-path/index.md

```diff
@@ -2,15 +2,15 @@ project_path: /web/_project.yaml
 book_path: /web/fundamentals/_book.yaml
 description: Ottimizzazione del percorso di rendering critico attraverso l'assegnazione di priorità alla visualizzazione del contenuto che è correlato all'azione principale che l'utente vuole effettuare sulla pagina.
 
-{# wf_updated_on: 2014-04-27 #}
+{# wf_updated_on: 2017-07-14 #}
 {# wf_published_on: 2014-03-31 #}
 
 # Percorso di rendering critico {: .page-title }
 
 {% include "web/_shared/contributors/ilyagrigorik.html" %}
 
 
-L'ottimizzazione del percorso di rendering critico è cruciale per il miglioramento delle prestazioni delle nostre pagine: il nostro scopo è assegnare priorità e visualizzare il contenuto relativo all'azione principale che l'utente vuole eseguire sulla pagina.
+_L'ottimizzazione del percorso di rendering critico_ è cruciale per il miglioramento delle prestazioni delle nostre pagine: il nostro scopo è assegnare priorità e visualizzare il contenuto relativo all'azione principale che l'utente vuole eseguire sulla pagina.
```

Comment:
I don't see this segment in the source file.

Reply:
You are right. It was on a previous version of `src/content/en/fundamentals/performance/critical-rendering-path/index.md`.